src/lib/sessionStorage.ts

```diff
@@ -6,6 +6,7 @@
 
 import type { ChatSavedPosition } from './appManagers/appImManager';
 import type { State } from './appManagers/appStateManager';
+import type { AppDraftsManager } from './appManagers/appDraftsManager';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
import type { AppDraftsManager } from "./appManagers/appDraftsManager";
```